.github/workflows/playwright-actions.yml

```diff
@@ -110,6 +110,22 @@ jobs:
           fi
           echo "PEM certificate is valid"
 
+      - name: Check CDN certificate expiry
+        working-directory: content-sources-backend
+        run: |
+          # Fail if certificate is expired
+          NOT_AFTER=$(openssl x509 -in ./configs/cdn.pem -noout -enddate | cut -d= -f2)
+          NOT_AFTER_EPOCH=$(date -d "$NOT_AFTER" +%s)
+          CURRENT_EPOCH=$(date +%s)
+          DAYS_UNTIL_EXPIRY=$(( (NOT_AFTER_EPOCH - CURRENT_EPOCH) / 86400 ))
+
+          if [[ $DAYS_UNTIL_EXPIRY -lt 0 ]]; then
```

Comment:
we might want to check if DAYS_UNTIL_EXPIRY is less than or equal to 0, so that this fails if it's set to expire today. it might also be helpful to show a warning message if the cert will expire within a week?

Reply:
Yes,
also shows how many days are left untill expiry
do we still want to mention specifically  when a week time is left ?

Reply:
if the job fails on the day it expires, that's probably enough :)